thewalrus/tests/test_quantum.py

```diff
@@ -1023,6 +1038,36 @@ def test_update_with_noise_coherent_value_error():
         update_probabilities_with_noise(noise_dists, probs)
 
 
+# @pytest.mark.parametrize("test_env_var", [None, "1", "2"])

```

Comment:
are you planning on leaving this here?

Reply:
Oh, no, good catch. Should've removed it. :facepalm: